dbt_checkpoint/check_exposure_has_meta_keys.py

```diff
@@ -0,0 +1,80 @@
+import argparse
+import os
+import time
+from pathlib import Path
+from typing import Any, Dict, Optional, Sequence
+
+from dbt_checkpoint.tracking import dbtCheckpointTracking
+from dbt_checkpoint.utils import (
+    JsonOpenError,
+    add_default_args,
+    add_meta_keys_args,
+    get_dbt_manifest,
+    get_exposures,
+    red,
+    yellow,
+)
+
+
+def has_meta_key(
```

Comment:
Could you make this function in a way that's agnostic to exposures, seeds, macros, sources etc, and then adding as a util?